thirdparty/build-thirdparty.sh

```diff
@@ -557,6 +557,20 @@ build_flatbuffers() {
   cp libflatbuffers.a $TP_LIB_DIR/libflatbuffers.a
 }
 
+build_brotli() {
+    check_if_source_exist $BROTLI_SOURCE
+    cd $TP_SOURCE_DIR/$BROTLI_SOURCE
+    mkdir -p $BUILD_DIR
+    cd $BUILD_DIR
+    rm -rf CMakeCache.txt CMakeFiles/
```

Comment:
thirdparty is supposed to be built from a brand new env. this clean up is not necessary.